test/verify/check-docker

```diff
@@ -261,6 +261,77 @@ CMD ["/listen-on-port.sh", "%(port)d", "%(message)s"]
 
         self.del_container_from_details("PROBE2")
 
+    def testVolumesAndEnvironment(self):
+        b = self.browser
+        m = self.machine
+
+        m.execute("systemctl start docker")
+
+        self.login_and_go("/docker")
+        b.wait_in_text("#containers-images", "busybox:latest")
+
+        # Select to view all running containers
+        b.wait_visible("#containers-containers-filter button")
+        b.click("#containers-containers-filter button")
+        b.click("#containers-containers-filter li a[value='all']")
+
+        # Create an updated image, expose a port
+        m.execute("mkdir /var/tmp/container-probe")
+        m.upload(["verify/files/listen-on-port.sh"], "/var/tmp/container-probe")
+        m.execute("""echo -e '
+FROM busybox
+ENV zero=GGG
+CMD ["/bin/sh"]
+' > /var/tmp/container-probe/Dockerfile""")
+        image_name = "test/container-probe"
+        m.execute("docker build -t %s /var/tmp/container-probe" % (image_name))
+        m.execute("rm -rf /var/tmp/container-probe")
+
+        # Wait for it to appear
+        b.wait_visible("#containers-images")
+        b.wait_in_text("#containers-images", image_name)
+
+        # Run it and expose additional port via the ui
```

Comment:
old comment, I suggest:
`# Run it and add another environment variable via the ui`